content/docs/how-we-work/local-tools.md

```diff
@@ -20,6 +20,31 @@ PHPStorm is perfect for working with core PHP and PHP frameworks such as Drupal,
 
 Visual Studio Code is a free and open-source editor that supports a broad range of programming languages. Visual Studio Code features a rich set of tools that enhance our productivity, such as an integrated terminal, Git integration, and IntelliSense, which provides real-time suggestions and code completion.
 
+#### Extensions/Plugins
+
+Like choosing an IDE that works for you is important, it is also equally important you have the necessary extensions/plugins that works well with your IDE and caters to your development workflow.
+
+As a developer some of the most common tasks done are,
+
+- using GitHub UI to interact with Pull requests (PRs) created by peers like review, merging or closing
+- updating the ticket status and progress in a tool like JIRA (there are other tools as well however we use JIRA at Axelerant).
```

Comment:
I think the phrase in parenthesis is unnecessary and makes this line longer to read.

Reply:
In fact, why not put these bullet points separately in the relevant headings below? A one-line introduction to this section is sufficient.

Reply:
Updated in 3c4b84d3faf44b8fc75227818bb31998aca02552